locale/de_DE/LC_MESSAGES/djangojs.po

```diff
@@ -1365,6 +1292,18 @@ msgstr ""
 "Weitere Informationen finden Sie in der Datenschutzerklärung von Vimeo "
 "unter: "
 
+#~ msgid "Blocked by moderator on"
+#~ msgstr "Von Moderation gesperrt"
+
+#~ msgid "Submit"
+#~ msgstr "Einreichen"
+
+#~ msgid "Other"
+#~ msgstr "Sonstige"
+
+#~ msgid "New answer"
+#~ msgstr "Neue Antwort"
+
```

Comment:
Is this that bug again or are they really deleted?

Reply:
I don't know, there were string changes, but we should also probably update a4 again. 
But let's only update the django.po files for now and we are safe and can handle everything a4 later!